subjects/devops/cloudPress/README.md

```diff
@@ -0,0 +1,84 @@
+## CloudPress
+
+![Cloud Deployment](./resources/architecture.png)
+
+### Objective
+
+CloudPress is a project designed to provide hands-on experience in deploying a WordPress website with MariaDB on the Amazon Web Services (AWS) cloud platform. The goal is to utilize Terraform for infrastructure creation and Ansible for application installation, resulting in a production-ready setup.
```

Comment:
I would add Ngnix here as well
`... deploying a WordPress website with MariaDB and Nginx ...`

Reply:
done